org.eclipse.winery.frontends/package.json

```diff
@@ -56,6 +56,7 @@
         "ngx-chips": "1.9.7",
         "ngx-pagination": "3.2.0",
         "ngx-toastr": "9.0.2",
+        "node-sass": "^7.0.1",
```

Comment:
Was the change intentional?

Reply:
No. Both, this and the changes in pom.xml were done automaticaly by some updates I think. I will revert both changes